scripts/histmakers/mz_wlike_with_mu_eta_pt.py

```diff
@@ -295,7 +295,7 @@
     # add usePseudoSmoothing=True for tests with Asimov
     muon_efficiency_helper, muon_efficiency_helper_syst, muon_efficiency_helper_stat = (
         muon_efficiencies_binned.make_muon_efficiency_helpers_binned(
-            filename=args.sfFile, era=era, max_pt=axis_pt.edges[-1], is_w_like=True
+            filename=args.sfFile, era=era, max_pt=axis_pt.edges[-1], is_z=True
```

Comment:
For consistency you should change this module argument too in wremnants/, even though we don't use anymore the binned SF